frontend/talentsearch/resources/js/components/ExperienceAccordion/AnyExperience.tsx

```diff
@@ -0,0 +1,18 @@
+import {
```

Comment:
Hmm, I think it's cool that you factored this out for reuse.  I'm not sure leaving it here is the best spot in the file tree but I can't think of a better spot.  Maybe someone else has an idea.

Reply:
Maybe moving it out of /components and into a helper file instead? ExperienceUtil.ts ?